From: Matthias Clasen <mclasen@redhat.com>
Date: Tue, 15 Jan 2019 23:02:31 -0500
Subject: Fix xml syntax in org.freedesktop.portal.Flatpak.xml

A bit concerning that this does not break the build.
It did break the docs build in xdg-desktop-portal.

Closes: #2602
Approved by: mwleeds
Applied-upstream: 1.1.4, commit:f28e8af9680074e1a6b4e6c90cb337fe2d52f26e
---
 data/org.freedesktop.portal.Flatpak.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/org.freedesktop.portal.Flatpak.xml b/data/org.freedesktop.portal.Flatpak.xml
index 9bddab0..bcb2d3d 100644
--- a/data/org.freedesktop.portal.Flatpak.xml
+++ b/data/org.freedesktop.portal.Flatpak.xml
@@ -111,7 +111,6 @@
                The files must be in the <filename>sandbox</filename> subdirectory of
                the instance directory (i.e. <filename>~/.var/app/$APP_ID/sandbox</filename>).
              </para></listitem>
-             </para></listitem>
            </varlistentry>
          </variablelist>
 
